Quotient/connection.h

```diff
@@ -771,7 +771,7 @@ public Q_SLOTS:
     //! \sa callApi
     void requestFailed(Quotient::BaseJob* request);
 
-    //! \brief A network request (job) failed due to network problems
+    //! \brief A network request (job) started by callAPI() failed due to network problems
```

Comment:
Not necessarily; you can equally create a job object and then pass it to `Connection::run()`, and the same `networkError()` will be emitted if it faces network problems.
```suggestion
    //! \brief A network request (job) failed due to network problems
```

Reply:
Sure, I've updated the comment. I was just copying what was written down for requestFailed(), even though it does appear to have that same behavior of also firing for jobs submitted via run().